### PR TITLE
docs: Add app showcase to landing page

### PR DESCRIPTION
**Pull request by marek-mihok**
This PR adds a humble showcase of what the Wave is capable of to our landing page:

https://user-images.githubusercontent.com/23740173/206700398-72c0eb75-fdf3-4470-ae1a-712936a285b1.mov

### CODE REVIEW

File: website/src/css/lightbox_styles.module.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/* stylelint-disable docusaurus/copyright-header */
```
Comment (mturoci) [inline website/src/css/lightbox_styles.module.css:1]: What's this?

File: website/src/pages/index.js [hunk 2]
```diff
@@ -127,8 +136,29 @@ function Home() {
             </div>
           </section>
         )}
+        {showcase && showcase.apps.length > 0 && (
+          <section className='container'>
+            <div className={clsx('row', styles.showcaseTitleContainer)} >
+              <div className={styles.divider} style={{ marginRight: 10 }} />
+              <h1 className={styles.showcaseTitle}>{`See what you can build\nwith Wave`}</h1>
```
Comment (mturoci) [inline website/src/pages/index.js:143]: There should be just a single `h1` per page in general. Please adjust the rest of the headings accordingly.
```suggestion
              <h2 className={styles.showcaseTitle}>What can Wave achieve?</h2>
```

File: website/src/pages/index.js [hunk 3]
```diff
@@ -127,8 +136,29 @@ function Home() {
...
+              <div className={styles.divider} style={{ marginLeft: 10 }} />
```
Comment (mturoci) [inline website/src/pages/index.js:144]: Let's also add a note under the h2:

The following are just a few of hundreds of Wave apps built within H2O.ai. The most mind-blowing thing is that all **presented apps were built by our data science and machine learning people**, no software engineer necessary.

File: website/src/pages/index.js [hunk 4]
```diff
@@ -127,8 +136,29 @@ function Home() {
...
+            </div>
+            <div className={clsx('row', styles.showcaseContainer)}>
+              {showcase.apps.map(({ title, description, images }, idx) =>
+                <div className={styles.showcaseRow} key={`row-${idx}`}>
```
Comment (mturoci) [inline website/src/pages/index.js:148]: nit: This should be enough

```suggestion
                <div className={styles.showcaseRow} key={idx}>
```

**Review by mturoci — CHANGES_REQUESTED** [review]
Looks great @marek-mihok and was also done pretty fast. Good job! Just some minor adjustments.

File: website/src/pages/index.js (hunk 3 above)
Comment (marek-mihok) [reply website/src/pages/index.js:144]: Good idea. What do you mean by note? Just simple `<p>` tag or does it need some special formatting?

Comment (mturoci) [reply website/src/pages/index.js:144]: `p` is fine

File: website/src/css/lightbox_styles.module.css (hunk 1 above)
Comment (marek-mihok) [reply website/src/css/lightbox_styles.module.css]: I was getting 'could not resolve module...' warning while importing a stylesheet into the `lightbox.jsx`. This seemed to help, but now I see the issue must have been somewhere else. I'm removing this line.

**Review by mturoci — APPROVED** [review]
Thanks @marek-mihok. Looks great!

**Pull request merged**